doc/source/index.rst

```diff
@@ -31,86 +31,99 @@ ICESat-2 datasets to enable scientific discovery.
 To further enhance data discovery, we have developed the QUEST module to facilitate querying of ICESat-2 data and complimentary Argo oceanographic data, with additional dataset support expected in the future.
 
 
-.. panels::
-    :card: + intro-card text-center
-    :column: col-lg-4 col-md-4 col-sm-6 col-xs-12 p-2
-    :img-top-cls: pl-2 pr-2 pt-2 pb-2
```

Comment:
Can we re-add the title centering (and perhaps title text background)? The updated version looks slightly less polished to me and I think that's why...

Reply:
I agree, it does, good call. I will give it a shot!

Reply:
7a423a8

I think the only difference I haven't accounted for is the grey background on the headers of the cards, but thought perhaps that wouldn't be worth the time. Let me know what you think!

Reply:
Thanks! It looks great.

> I think the only difference I haven't accounted for is the grey background on the headers of the cards, but thought perhaps that wouldn't be worth the time.

Probably true. Not a problem for today.